diff --git a/src/datamodel_code_generator/_format_types.py b/src/datamodel_code_generator/_format_types.py
--- a/src/datamodel_code_generator/_format_types.py
+++ b/src/datamodel_code_generator/_format_types.py
@@ -28,6 +28,7 @@ class PythonVersion(Enum):
     PY_311 = "3.11"
     PY_312 = "3.12"
     PY_313 = "3.13"
     PY_314 = "3.14"
+    PY_315 = "3.15"
 
     @cached_property
     def version_key(self) -> tuple[int, int]:
